Volume/Views/Onboarding/PublicationDetail/PublicationDetail.swift

```diff
@@ -14,22 +14,35 @@ import SwiftUI
 struct PublicationDetail: View {
     @Environment(\.presentationMode) private var presentationMode: Binding<PresentationMode>
     @GestureState private var dragOffset = CGSize.zero
-    @State private var cancellableQuery: AnyCancellable?
+    @State private var cancellableIDQuery: AnyCancellable?
+    @State private var cancellableArticlesQuery: AnyCancellable?
     @State private var state: PublicationDetailState = .loading
 
     let navigationSource: NavigationSource
     let publication: Publication
 
     private func fetch() {
-        cancellableQuery = Network.shared.publisher(for: GetArticlesByPublicationIdQuery(id: publication.id))
+        cancellableIDQuery = Network.shared.publisher(for: GetPublicationBySlugQuery(slug: publication.slug))
+            .map { $0.publication.map{ $0.fragments.publicationFields.id }! }
+            .sink {
+                if case let .failure(error) = $0 {
+                    print("GetPublicationBySlugQuery failed: \(error)")
+                }
+            } receiveValue: {
```

Comment:
I've never seen this syntax, what is going on here?

Reply:
Closures as parameters must be named if they aren't the first parameter